File: solutions/javascript/004-anagram/solution.test.mjs
import test from "node:test";
import assert from "node:assert/strict";
import { isAnagram } from "./solution.mjs";

test("anagrams", () => {
  assert.equal(isAnagram("listen", "silent"), true);
  assert.equal(isAnagram("Listen", "Silent"), true);
  assert.equal(isAnagram("A gentleman", "Elegant man"), true);
  assert.equal(isAnagram("rail safety", "fairy tales"), true);
  assert.equal(isAnagram("hello", "world"), false);
  assert.equal(isAnagram("abc", "ab"), false);
});

File: solutions/python/004-anagram/solution_test.py
from solution import is_anagram

assert is_anagram("listen", "silent") is True
assert is_anagram("Listen", "Silent") is True
assert is_anagram("A gentleman", "Elegant man") is True
assert is_anagram("rail safety", "fairy tales") is True
assert is_anagram("hello", "world") is False
assert is_anagram("abc", "ab") is False
print("ok")
